evaluation-tasks/.gitignore

```diff
@@ -0,0 +1,6 @@
+# Working directory for luigi pipelines
```

Comment:
What do you think about having just one top-level gitignore? I've never used several but I'm open to it.

Reply:
+1